### DISCUSSION TITLE
Problems with the enterprise edition connection number limit

### DISCUSSION THREAD

**Discussion by SizerSong** [ONLYOFFICE/DocumentServer#1784] [Q&A] [answered]
1. The "Number of connections". Whether it's the total number of people using onlyoffice plug-ins or the number of people editing the same document?
2. Is there any way to increase the connection limit in community edition?

**Comment by ShockwaveNN** [comment, marked as answer]
> onlyoffice plug-ins

What do you mean by onlyoffice plug-ins? Connectors for nextcloud or other DMS?
If so - no, connectors is not count at all

> editing the same document?

Not the same, but any document


> Is there any way to increase the connection limit in community edition?

Buy license )

**Comment by SizerSong** [comment]
I get it, thanks for your reply :)
